### PR TITLE
Org update should throw error if not found

### PR DESCRIPTION
**Pull request by utkarshcmu**

### CODE REVIEW

File: pkg/services/sqlstore/org.go [hunk 1]
```diff
@@ -122,6 +122,13 @@ func CreateOrg(cmd *m.CreateOrgCommand) error {
 func UpdateOrg(cmd *m.UpdateOrgCommand) error {
 	return inTransaction2(func(sess *session) error {
 
+		orgQuery := &m.GetOrgByIdQuery{Id: cmd.OrgId}
+
+		orgDoesntExists := GetOrgById(orgQuery)
```
Comment (torkelo) [inline pkg/services/sqlstore/org.go]: think it's better to just check if the update affected any rows

Comment (torkelo) [inline pkg/services/sqlstore/org.go]: Used when updating dasbhoard , https://github.com/grafana/grafana/blob/d670f1b959fbbb673a669dd6ddb49326ff7e5ae4/pkg/services/sqlstore/dashboard.go#L80

Comment (bergquist) [inline pkg/services/sqlstore/org.go]: Relaying on affected rows might not work with all databases.
https://golang.org/pkg/database/sql/#Result

Comment (torkelo) [inline pkg/services/sqlstore/org.go]: Do we support want (or want to) support one that does not support it?

**Comment by utkarshcmu** [comment]
Used affected rows as suggested by @torkelo

**Comment by torkelo** [comment]
Awesome!

File: pkg/services/sqlstore/org.go (hunk 1 above)
Comment (bergquist) [inline pkg/services/sqlstore/org.go]: How do we know which databases support it? 

I guess we could write a uni test that confirms it and make sure to run it on all databases that we support. (as we should regardless) :)

**Pull request merged**